chapters/08-erros-e-manipulacao-de-erros.md

```diff
@@ -6,11 +6,11 @@
 > Yuan-Ma havia escrito um pequeno programa onde utilizou muitas variáveis globais e atalhos que faziam a qualidade do seu código inferior. Lendo o programa, um estudante perguntou: “Você nos avisou para não usar essas técnicas e mesmo assim as encontro no seu programa. Como pode isso?”. O mestre respondeu: “Não há necessidade de se buscar uma mangueira de água quando a casa não está em chamas.”
 > — Master Yuan-Ma, The Book of Programming
 
-Um programa são pensamentos cristalizados. Às vezes esses pensamentos são confusos. Outras vezes os erros são introduzidos ao converter esse pensamento em código. De qualquer forma o resultado será um programa falho.
+Um programa é um pensamento “cristalizado”. Algumas vezes, esses pensamentos são confusos e erros podem ser inseridos quando convertemos pensamentos em código, resultando em um programa com falhas.
 
-Falhas em um programa são considerados como bugs. Bugs podem ser erros causados pelo programador ou problemas em outros sistemas na qual o programa interage. Alguns erros podem serem detectados facilmente enquanto outros são extremamente sutis e pode permanecer escondido em um sistema por anos.
+Falhas em um programa são normalmente chamadas de “bugs”, e podem ser causadas por erros inseridos pelo programador ou problemas em outros sistemas que a aplicação interage. Alguns *bugs* são imediatamente aparentes, enquanto outros são sutis e podem ficar escondidos em um sistema por anos.
```

Comment:
Boa! talvez nesse caso seja até melhor manter tudo em itálico, já que é um termo em inglês. nesse caso as aspas não fazem muito sentido.